cfgov/unprocessed/css/organisms/video-player.less

```diff
@@ -200,7 +205,7 @@ button.o-video-player_play-btn,
         position: absolute;
         top: 0;
     }
-} );
+});
```

Comment:
This should be reverted. The opening line for this `.respond-to-min( @bp-sm-min, {` has a space in the parentheses (and we generally pad around parens).